tests/ci-helper.test.ts

```diff
@@ -888,6 +904,7 @@ test("handle push/comment too many commits fails", async () => {
     ci.setGHGetPRInfo(prInfo);
     ci.setGHGetPRComment(comment);
     ci.setGHGetGitHubUserInfo(user);
+    ci.setGHGetPRCommits(commits);
```

Comment:
It wasn't really forgotten.  It was more that it did not need to be mocked knowing the PR had no merges to affect the test results.  I agree it should have been mocked in retrospect.